packages/devtools_app/lib/src/logging/checkout_box.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:html_shim/html.dart' as html;
+
+import '../../src/ui/html_elements.dart';
+import 'config.dart';
+
+class ToolBarCheckbox {
```

Comment:
Add the word Html to the class name as done elsewhere in the dart:html version of the app. We'll need to create a Flutter version of this class in the port of the app to flutter.